### DISCUSSION TITLE
Date picker without the default input fields

### DISCUSSION THREAD

**Discussion by yashsolanki22** [rsuite/rsuite#3999] [Q&A]
Hello team, may i know if rsuite supports rendering only the datepicker panel without rendering the default input fields so i can just directly set the open state to always true and use the panel in some popover or dialog box.

<img width="2054" alt="image" src="https://github.com/user-attachments/assets/3248675f-ed9b-431a-b4bb-6a803eb0eb54">

**Comment by simonguo** [comment]
You can try the Calendar component, which is the basic component of DatePicker and does not include an input box.
https://rsuitejs.com/components/calendar/#compact

**Reply by yashsolanki22** [reply]
thanks for the heads up. I'm wondering if calender supports the option to disable past dates or not?

**Reply by simonguo** [reply]
Currently not supported, but this feature will be considered in the future.

**Reply by yashsolanki22** [reply]
the default datepicker and range picker works fine but the only constrain was the placeholder value which happens to be auto applied when the input comes into the focus or click i.e. dd/MM/yyyy or any placeholder. I've tried to find the solution for this because i do not want any placeholder value but certainly unable to do so. If you have suggestion please let me know
Thanks!
